master-branch does not compile

```
[ERROR] /git/docker-java/src/main/java/com/github/dockerjava/netty/NettyDockerCmdExecFactory.java:[322,46] dockerClientConfig has private access in com.github.dockerjava.core.AbstractDockerCmdExecFactory
[INFO] 1 error
```

--- a/src/main/java/com/github/dockerjava/netty/NettyDockerCmdExecFactory.java
+++ b/src/main/java/com/github/dockerjava/netty/NettyDockerCmdExecFactory.java
@@ -319,6 +319,7 @@ public class NettyDockerCmdExecFactory extends AbstractDockerCmdExecFactory impl
             if (channel == null || !channel.isActive() || alreadyTimedOut) {
                 return;
             }
+            DockerClientConfig dockerClientConfig = getDockerClientConfig();
             final Object dockerAPIEndpoint = dockerClientConfig.getDockerHost();
             final String msg = "Read timed out: No data received within " + readTimeout
                     + "ms.  Perhaps the docker API (" + dockerAPIEndpoint

